targets/xorg

```diff
@@ -97,7 +97,7 @@ fi
 
 # On Freon, we need crazy xorg hacks
 if [ -n "$freon" ]; then
-    compile freon '-ldl -ldrm -I/usr/include/libdrm' so libdrm-dev
+    compile freon '-ldl -ldrm -I/usr/include/libdrm' so openSUSE=libdrm-devel,libdrm-dev
```

Comment:
same goes for devel vs. dev, I guess.  You don't have to make any changes, though.